### ISSUE TITLE
please search for Markdown-Files recursive (including Subfolders)

### ISSUE THREAD

**Issue by benpla** [daeh/zotero-obsidian-citations#2]
It would be very nice if the path to the Markdown notes would be searched including the subfolders. 

To explain my reasons:
I collect the literature notes in a folder, but I divide them by project, because I have several hundred citations per project and in this way I can quickly find all the literature for the project without searching.

**Comment by daeh** [comment]
@benpla update the addons (`Tools -> Add-ons -> Check for Updates`) and let me know if v0.0.4 works for you

**Comment by benpla** [comment]
Many Thanks.
v0.0.4 works as expected and find .MD files in subfolders.

v0.0.5 breaks this: same preferences, but error message:
![Screenshot](https://user-images.githubusercontent.com/38682514/148902149-f11143f4-2527-46a2-98a7-ccaed8d84e48.png)

Preferences Regex: 
```
^- cloud::.*items\/(\w*)
```

Example of MD-File `@AlKhalifaAlRazgan_2016_IndoorGuidanceForVisuallyImpaired.md`:
[@AlKhalifaAlRazgan_2016_IndoorGuidanceForVisuallyImpaired.md](https://github.com/daeh/zotero-obsidian-citations/files/7844680/%40AlKhalifaAlRazgan_2016_IndoorGuidanceForVisuallyImpaired.md)

Snippet for triggering the regex in the .MD-File:
```
- cloud:: http://zotero.org/users/4847583/items/XYJ7WD6V
```

**Comment by daeh** [comment]
I crossed a wire and switched to variable assignments. should work now. Fixed by 809cb0b4250e53fc213693bba85c422b75e76270

**Comment by benpla** [comment]
I just tested it: it works great!!!
Your addon also detected 5 MD files that have no Zotero entry. I must have deleted once too much.... This way your plugin helps me to keep order and that's great.

I find the JSON-log files very nice, so I can track in detail which entry is missing and track down the cause.

Thanks a lot for the work!
